packages/rke2-coredns/package.yaml

```
@@ -1,4 +1,4 @@
-url: https://github.com/coredns/helm/releases/download/coredns-1.16.3/coredns-1.16.3.tgz
-packageVersion: 02
+url: https://github.com/coredns/helm/releases/download/coredns-1.16.4/coredns-1.16.4.tgz
+packageVersion: 03
```

You can drop this back down to 0 when bumping the upstream chart version; the package version only needs to be incremented if changing the local customizations without bumping the upstream chart version.
```suggestion
packageVersion: 00
```

---

Ok. Fixed